types/database/schema/ChannelList.types.ts

```diff
@@ -41,7 +41,8 @@ export type BetterSocialChannelType =
   | 'ANON_PM'
   | 'POST_NOTIFICATION'
   | 'ANON_POST_NOTIFICATION'
-  | 'TOPIC';
+  | 'TOPIC'
+  | 'ANON_TOPIC';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of the 'ANON_TOPIC' to the `BetterSocialChannelType` enum is a good change. It allows for more specific channel types and can improve the handling of different channel types in your application. However, ensure that this new type is properly handled wherever `BetterSocialChannelType` is used throughout your codebase.
```diff
   | 'ANON_PM'
   | 'POST_NOTIFICATION'
   | 'ANON_POST_NOTIFICATION'
+  | 'TOPIC'
+  | 'ANON_TOPIC';
 
 export interface ChannelList {
   id: string;
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->